aptos-move/aptos-native-interface/src/context.rs

```diff
@@ -123,4 +123,9 @@ impl<'a, 'b, 'c, 'd> SafeNativeContext<'a, 'b, 'c, 'd> {
     pub fn aggregator_snapshots_enabled(&self) -> bool {
         self.get_feature_flags().is_aggregator_snapshots_enabled()
     }
+
+    /// TODO: decide if we want to have two flags or merge with aggregator_snapshots_enabled.
```

Comment:
we are introducing new native functions that need to be gated. 

since aggregator_snapshots_enabled was never enabled, we can either:
- decide not to enable it before v1.8, and combine aggregator_api_enabled into it. 
- if we want to enable in point release for v1.7, we would need to add a new flag instead.